lib/hammer_cli_foreman/compute_resource/vmware.rb

```diff
@@ -9,13 +9,13 @@ def name
 
       def compute_attributes
         [
-          ['cpus',                 _('CPU count')],
-          ['corespersocket',       _('Number of cores per socket (applicable to hardware versions < 10 only)')],
-          ['memory_mb',            _('Integer number, amount of memory in MB')],
+          ['cpus',                 _('CPU count'), { bold: true }],
+          ['corespersocket',       _('Number of cores per socket (applicable to hardware versions < 10 only)'), { bold: true }],
+          ['memory_mb',            _('Integer number, amount of memory in MB'), { bold: true }],
           ['firmware',             'automatic/bios/efi'],
```

Comment:
Could we put the `firmware` after the required attributes? Now it feels bit weird to be here.
Or sort it alphabeticaly, but now it feels bit messy ordered :)